Tests/OutputFunctions.asm

```diff
@@ -42,6 +43,16 @@ AdvanceVramHlToNextLine:
     ld      a,32
     jr      AdvanceVramHlByAChars
 
+; advances HL by 32 (works as "next line" in ZX attributes VRAM area)
+; modifies: A and HL
+AdvanceAttrHlToNextLine:
```

Comment:
Assume you're not using the Next specific opcode ADD HL, 32 here so you're not reliant on Next stuff, given that it's the Next under test?

Reply:
Yes, I'm intentionally using only classic Z80 instructions in these support functions, to make the "Next bootstrapping" process for emulator authors somewhat easier. My intent is to produce few "base" tests which can do *something* already on minimal ZX48 emulator, adding Next subsystems in steps (one of the possible "base" tests may exercise Z80N instructions).

For the non-base tests I think full usage of Z80N is reasonable, actually preferred, that way the whole "next" system will be tested.